aospy/model.py

```diff
@@ -10,8 +10,14 @@
 from . import utils
 
 # Dask must use its serial scheduler if computations are to be performed
-# in parallel using multiprocess
-dask.set_options(get=dask.async.get_sync)
```

Comment:
@spencerahill good catch!  It's possible this line may not even be needed anymore now that we use `dask.bag` for parallel computations.  Could you see if the test suite passes with it removed?

See [this line](https://github.com/spencerahill/aospy/pull/172/files#diff-c1bc79d9c0310c1c1c15a5943463101cL14) in #172, where this code was already removed from `data_loader.py`.  Sorry if I missed this there.